src/libs/ontuml2gufo/constants.ts

```diff
@@ -0,0 +1,76 @@
+export const RelationsAsPredicate = ['instantiation', 'derivation'];
+
+export const HideObjectPropertyCreationList = [
+  'characterization',
+  'comparative',
+  'componentOf',
+  'creation',
+  'derivation',
+  'externalDependence',
+  'historicalDependence',
+  'instantiation',
+  'manifestation',
+  'material',
+  'mediation',
+  'memberOf',
+  'participation',
+  'participational',
+  'subCollectionOf',
+  'subQuantityOf',
+  'termination',
+];
+
+export const HideReadOnlyObjectPropertyCreationList = [
+  'componentOf',
+  'memberOf',
+  'subCollectionOf',
+  'subQuantityOf',
+];
+
+export const RelationsInverted = ['creation', 'termination', 'participation'];
+
+export const NormalRelationStereotypeMapping = {
+  characterization: 'inheresIn',
+  comparative: 'comparativeRelationshipType',
+  componentOf: 'isComponentOf',
+  creation: 'wasCreatedIn',
+  derivation: 'isDerivedFrom',
+  externalDependence: 'externallyDependsOn',
+  historicalDependence: 'historicallyDependsOn',
+  instantiation: 'categorizes',
+  manifestation: 'manifestedIn',
+  material: 'materialRelationshipType',
+  mediation: 'mediates',
+  memberOf: 'isCollectionMemberOf',
+  participation: 'participatedIn',
+  participational: 'isEventProperPartOf',
+  subCollectionOf: 'isSubCollectionOf',
+  subQuantityOf: 'isSubQuantityOf',
+  termination: 'wasTerminatedIn',
+  isProperPartOf: 'isProperPartOf',
+};
+
+export const IgonoredInverseRelations = ['derivation', 'instantiation'];
```

Comment:
You are creating `gufo:comparativeRelationshipType` and `gufo:materialRelationshipType`.
They should never be created!